sdk/src/main/scala/com/horizen/api/http/SidechainSubmitterApiRoute.scala

```diff
@@ -43,6 +43,10 @@ case class SidechainSubmitterApiRoute[
   extends SidechainApiRoute[TX, H, PM, FPI, NH, NS, NW, NP, NV]
   with ApiDirectives
 {
+
+  // TODO check this:
+  //  sidechainNodeViewHolderRef is not used, but we keep it since it is defined as an abstract member of the base class
+
```

Comment:
we may remove it and extend `ApiRoute` trait.
Or keep it as is, just in case in the future it will be used. (I vote to keep it)

Reply:
Ok, I kept it. Commented code removed